scripts/codestyle/travis_uncrustify.sh

```diff
@@ -0,0 +1,7 @@
+echo $PWD
+if ! $TRAVIS_PULL_REQUEST; then
+	bash ./run_uncrustify.sh -v $(git rev-parse HEAD~1) $(git rev-parse HEAD)
```

Comment:
May be, or just a list with path to scan as a variable in script.
I think it should be include list (src), not exclude list (third-party, scripts,...)